tests/test_engine.py

```diff
@@ -167,6 +204,7 @@ def test_transaction_rollback(self):
         self.successResultOf(trx.rollback())
         rows = self.execute_fetchall(conn, tbl.select())
         assert len(rows) == 0
+        conn.execute(DropTable(tbl))
```

Comment:
Different tests probably shouldn't be sharing the same database and trying to clean it up. If they are trying to clean it up, they should be doing it in a [`cleanup`](https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.addCleanup), not in-line at the end of the test.